cvat-ui/src/components/job-item/job-item.tsx

```diff
@@ -125,35 +129,31 @@ function JobItem(props: Props): JSX.Element {
     const frameCountPercentRepresentation = frameCountPercent === '0' ? '<1' : frameCountPercent;
     const jobName = `Job #${job.id}`;
 
-    let childJobs: Job[] = [];
-    if (task.consensusEnabled) {
-        childJobs = task.jobs.filter((eachJob: Job) => eachJob.parentJobId === job.id).reverse();
-    }
-    const consensusJobViews: React.JSX.Element[] = childJobs.map((eachJob: Job) => (
+    const childJobViews: React.JSX.Element[] = childJobs ? childJobs.map((eachJob: Job) => (
         <JobItem key={eachJob.id} job={eachJob} task={task} onJobUpdate={onJobUpdate} />
-    ));
-
-    const regularJobViewUncollapse = useSelector((state: CombinedState) => state.jobs.regularJobViewUncollapse);
-    const regularJobUncollapsed = regularJobViewUncollapse[job.id];
-    const handleCollapseChange = async (): Promise<void> => {
-        await dispatch(collapseRegularJob(job.id, !regularJobUncollapsed));
-    };
```

Comment:
@klakhov, somewhere along the changed lines here we lost a feature to remember uncollapsed jobs on the task page, so that on the next page visit the same jobs were displayed. Could you please restore this functionality?